### ISSUE TITLE
[BUG]: Menus no longer working on Mobian

### ISSUE THREAD

**Issue by morgannunan** [martinrotter/rssguard#370]
#### Brief description of the issue.

I'm sure this is an edge case as far as you're concerned, but I use RSS Guard on my Linux mobile device, and the menus have become unresponsive. If I click on any main menu items, they will highlight, but nothing will open.

I have tried upgrading to 3.9, and down to 3.8.4. It works fine on my desktop, so it must be specific to ARM64 or to my Mobian installation.

If I use a Bluetooth mouse, I still can't click on the menu items. I have also tried removing all RSS Guard config files.

#### How to reproduce the bug?

1. Open Rss Guard on Mobian
2. Press any main menu buttons

#### What was the expected result?

Submenu will open

#### What actually happened?

Submenus don't open

#### Other information

* OS: Mobian (Debian) Bullseye
* Desktop Environment: PinePhone
* RSS Guard version: 3.8 and 3.9

**Comment by martinrotter** [comment]
@morgannunan 
Wait, so version 3.8.4 works fine and 3.9.0 is first version which has the bug?

**Comment by morgannunan** [comment]
No, it doesn't work for me for 3.8.4, either. It worked for me until about the start of the year. I  couldn't tell you exactly when it stopped working. Also, I was updating my phone pretty regularly, so it could be an issue with Mobian, or QT. I would like to keep using RSS Guard, though!

**Comment by martinrotter** [comment]
This cannot be caused directly by RSS Guard, it must be something with upstream libs. There weren't any menu-related code changes in RSS Guard for years.

Quick google search reveals that there seem to be quite a number of problems with Qt menus on ARM operating systems.

https://bugreports.qt.io/browse/QTBUG-75583

**Comment by martinrotter** [comment]
I pushed experimental fix and it should be compiled as development build for RSS Guard 4.x in 15 minutes. Feel free to test somehow. Unfortunately, I have no way to test RN on ARM.

The other thing you can try is to use "corner" button to access the menu (the small button in top/left corner with the "home" icon): 
![image](https://user-images.githubusercontent.com/1255302/109456624-87906f80-7a59-11eb-821b-041345046bc2.png)

**Comment by martinrotter** [comment]
So I am sorry, but this must be upstream bug. Does similar problems happen with other Qt apps?

**Comment by morgannunan** [comment]
@martinrotter, I appreciate you looking into this. I tried installing mixxx on my phone, which is Qt based, and the interface is unresponsive there, too. Also, I installed VLC before and I couldn't get it to work, either.

In the case of VLC, I believe it was popping up a tip on start that I couldn't see on the screen, but that was freezing the interface. Sending the esc key through the keyboard fixed it for me. Is it possible that RSS Guard is trying to pop up a modal when it loads?
